etc/repos.yaml

```diff
@@ -131,6 +131,7 @@ qserv: https://github.com/lsst/qserv.git
 palpy: https://github.com/lsst/palpy.git
 luaexpat: https://github.com/lsst-dm/legacy-luaexpat.git
 cat: https://github.com/lsst/cat.git
+sdm_schemas: https://github.com/lsst-dm/sdm_schemas.git
```

Comment:
Is sdm_schemas a renamed cat or a completely different git history? If it's a rename why can't it be renamed directly in the lsst org? Then you could add the entry and also forward `cat` to the new name.

Reply:
It's a renamed cat. I'm following the steps outlined in https://dmtn-027.lsst.io 

It suggests to put the renamed package in lsst-dm while testing & the first round of Jenkins, before merging the relevant changes and moving it to lsst org.